packages/augur-core/source/contracts/WarpSync.sol

```diff
@@ -70,6 +75,13 @@ contract WarpSync is IWarpSync, Initializable {
             return;
         }
 
+        // In order to periodically sweep interest we do so here which will result in a sweep at least on recurring basis where the sweeper is compensated.
+        uint256 _timestamp = block.timestamp;
```

Comment:
Should this be using Augur.time (or whatever it is called) rather than `block.timestamp`?  This would allow testing of the timing functionality.

Reply:
If this doesn't end up using Augur time contract, then consider getting rid of the local variable.  I'm not confident (double check) but I think `block.timestamp` is similar cost as reading from memory, so this local variable assignment may not be buying anything.

Reply:
Ah yea should use the controlled time